src/BugsnagServiceProvider.php

```diff
@@ -269,8 +269,8 @@ protected function setupCallbacks(Client $client, Container $app, array $config)
         if (!isset($config['user']) || $config['user']) {
             $client->registerCallback(new CustomUser(function () use ($app) {
                 if ($user = $app->auth->user()) {
-                    if (is_callable([$user, 'toArray'])) {
-                        return $user->toArray();
+                    if (is_callable([$user, 'attributesToArray'])) {
```

Comment:
This should actually be:

```php
if (method_exists($user, 'attributesToArray') && is_callable([$user, 'attributesToArray'])) {
    return $user->attributesToArray();
}
```

This was a bug in the original code too.

---

The reason here is that eloquent models have a `__call` method, so `is_callable` returns true even for methods that aren't really callable, because it just throws a runtime exception if not callable. We can't just have a `method_exists` check here either, because that also returns `true` for private methods, and we are only interested in ones we can actually call.